packages/geo/src/quad.key.ts

```diff
@@ -56,6 +61,45 @@ export const QuadKey = {
         return tileToBBOX(quadkeyToTile(quadKey));
     },
 
+    /**
+     * Convert a quadkey to a XYZ Tile location
+     * @param quadKey quadkey to convert
+     */
+    toTile(quadKey: string): Tile {
+        let x = 0;
+        let y = 0;
+        const z = quadKey.length;
+
+        for (let i = z; i > 0; i--) {
+            const mask = 1 << (i - 1);
+            const q = quadKey.charCodeAt(z - i);
+            if (q === CHAR_1) x |= mask;
+            if (q === CHAR_2) y |= mask;
+            if (q === CHAR_3) {
```

Comment:
Why using char code? I think this is actually slower than `quadkey[z-1]`

Reply:
Im pretty sure using array indexes on strings actually creates a new string, which is kinda slow, I'll benchmark it

Reply:
node 12

| Command | Mean [ms] | Min [ms] | Max [ms] | Relative |
|:---|---:|---:|---:|---:|
| `node tests/char.code.js` | 398.1 ± 5.8 | 391.1 | 407.4 | 1.0 |
| `node tests/array.index.js` | 447.0 ± 7.6 | 440.7 | 466.0 | 1.1 |

node 14

| Command | Mean [ms] | Min [ms] | Max [ms] | Relative |
|:---|---:|---:|---:|---:|
| `node tests/char.code.js` | 393.3 ± 9.4 | 383.4 | 427.2 | 1.0 |
| `node tests/array.index.js` | 560.6 ± 16.0 | 541.0 | 617.3 | 1.4 |